### DISCUSSION TITLE
Where is the username field on users collection

### DISCUSSION THREAD

**Discussion by gelomon** [pocketbase/pocketbase#6164] [Q&A] [answered]
Hello I have updated my pocketbase version from 0.22.18 to the latest n-1 0.23.11 and I have noticed that on users, the username field is now removed. is there a way to enable it or I should move on to use email to authenticate user?

Thank you

**Comment by chnfyi** [comment, marked as answer]
Hi, the default identity field is the email but you can configure any other unique field like _username_ (it must have a UNIQUE index).

To make it works you have to add the _username_ field to the _unique identity fields list_ that you'll find at _edit collection > options > auth methods > identity/password_.

```javascript
import PocketBase from 'pocketbase';

const pb = new PocketBase('http://127.0.0.1:8090');

...

const authData = await pb.collection("users").authWithPassword('username', '1234567890');
```

**Reply by gelomon** [reply]
Thanks for the quick response, here is what I have done:

![image](https://github.com/user-attachments/assets/034b0e8b-6b84-408a-af62-ab13a25cffaf)

is this correct?

**Reply by chnfyi** [reply]
Yes, see my edited answer to configure it properly.

**Reply by gelomon** [reply]
Thanks! updated the options to include username!
